README.md

```diff
@@ -1,30 +1,35 @@
 # lloesche/valheim-server Docker image
 ![Valheim](https://raw.githubusercontent.com/lloesche/valheim-server-docker/main/misc/Logo_valheim.png "Valheim")
 
-Valheim Server in a Docker Container  
+Valheim Server in a Docker Container (with optional [Valheim Plus](https://github.com/nxPublic/ValheimPlus) mod support)  
 
 # Basic Docker Usage
 
 The name of the Docker image is `lloesche/valheim-server`.
 
 Volume mount the server config directory to `/config` within the Docker container.
 
+If you would also like to enable Valheim Plus, please also volume mount the server files directory `/opt/valheim_dl` so that you can modify the mod's config file.
+
 If you have an existing world on a Windows system you can copy it from e.g.  
   `C:\Users\Lukas\AppData\LocalLow\IronGate\Valheim\worlds`  
 to e.g.  
   `$HOME/valheim-server-config/worlds`  
 and run the image with `$HOME/valheim-server-config` volume mounted to `/config` inside the container.
 
 ```
-$ mkdir -p $HOME/valheim-server-config/worlds
+$ mkdir -p $HOME/valheim-server/config/worlds
 # copy existing world
 $ docker run -d \
     --name valheim-server \
     -p 2456-2458:2456-2458/udp \
-    -v $HOME/valheim-server-config:/config \
+    -v $HOME/valheim-server/config:/config \
+    -v $HOME/valheim-server/files:/opt/valheim_dl \
     -e SERVER_NAME="My Server" \
-    -e WORLD_NAME="Neotopia" \
     -e SERVER_PASS="secret" \
+    -e WORLD_NAME="Neotopia" \
+    -e PUBLIC=1 \
+    -e VALHEIM_PLUS_ENABLED=true
```

Comment:
We should disable by default as Valheim Plus requires clients to have the mod installed as well (except for server only settings). People should intentionally enable the mod.

Reply:
By default, the container will not run with Valheim Plus unless explicitly asked to. I put this in the README as an example for people who want to just quickly get started but I can see how it can be confusing. Will remove it

Reply:
I figured, but I'd just change true to false. That way people see that the container supports V+ but don't accidentally enable it.